### PR TITLE
translate content/en/blog/_posts/2022-08-16-PSP-historical-context/index.md into Chinese

### PR DESCRIPTION
**Pull request by sarazqy**
translate content/en/blog/_posts/2022-08-16-PSP-historical-context/index.md into Chinese

fixes: https://github.com/kubernetes/website/issues/36440

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+并继续构建我们所知道的 PSP 准入。之后，进行了许多改进和修复，以构建最近 Kubernetes 版本的 PodSecurityPolicy 功能。
+
+
+<!--
+## The rise of Pod Security Admission 
+
+Despite the crucial issue it was trying to solve, PodSecurityPolicy presented
+some major flaws:
+-->
+## Pod 安全接入的兴起
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:155]:
```suggestion
## Pod 安全准入的兴起
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+  composability with a weak prioritization model, leading to unexpected
+  mutation priority. It made it really difficult to combine PSP with other
+  third-party admission controllers.
+- **Require security knowledge** - effective usage still requires an
+  understanding of Linux security primitives. e.g. MustRunAsNonRoot +
+  AllowPrivilegeEscalation.
+-->
+- **有缺陷的授权模式** - 如果用户在 PSP 上具有允许该 Pod 的**使用**动词，或者该 Pod 
+  的服务帐户在允许的 PSP 上具有**使用**权限，则用户可以创建一个 Pod。
+- **难以推广** - PSP失败关闭。也就是说，在没有策略的情况下，所有 Pod 都会被拒绝。
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:182]:
```suggestion
- **难以推广** - PSP 失败关闭。也就是说，在没有策略的情况下，所有 Pod 都会被拒绝。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+-->
+**作者：** Mahé Tardy (Quarkslab)
+
+<!--
+The PodSecurityPolicy (PSP) admission controller has been removed, as of
+Kubernetes v1.25. Its deprecation was announced and detailed in the blog post
+[PodSecurityPolicy Deprecation: Past, Present, and Future](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/),
+published for the Kubernetes v1.21 release.
+-->
+从 Kubernetes v1.25 开始，PodSecurityPolicy(PSP) 准入控制器已被删除。
```
[the remark this answers is not in the document]
Comment (Michelle951) [reply content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:26]: Why left this one?

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
+---
+layout: blog
+title: "PodSecurityPolicy: 历史背景"
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:3]:
```suggestion
title: "PodSecurityPolicy：历史背景"
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+-->
+**作者：** Mahé Tardy (Quarkslab)
+
+<!--
+The PodSecurityPolicy (PSP) admission controller has been removed, as of
+Kubernetes v1.25. Its deprecation was announced and detailed in the blog post
+[PodSecurityPolicy Deprecation: Past, Present, and Future](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/),
+published for the Kubernetes v1.21 release.
+-->
+从 Kubernetes v1.25 开始，PodSecurityPolicy (PSP) 准入控制器已被删除。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:26]:
```suggestion
从 Kubernetes v1.25 开始，PodSecurityPolicy (PSP) 准入控制器已被移除。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+fine-grained permissions on specific fields concerning the pod security settings
+as a built-in policy API. It acknowledged that cluster administrators and
+cluster users are usually not the same people, and that creating workloads in
+the form of a Pod or any resource that will create a Pod should not equal being
+"root on the cluster". It could also encourage best practices by configuring
+more secure defaults through mutation and decoupling low-level Linux security
+decisions from the deployment process.
+-->
+PodSecurityPolicy 与其他专门的准入控制插件一样，在与 Pod 安全设置有关的特定字段上提供细粒度权限，
+作为内置策略 API。它承认集群管理员和集群用户通常不是同一个人，并且以 Pod 或任何将创建 Pod 的资源的形式创建工作负载不应该等同于“集群上的根”。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:48]:
```suggestion
作为内置策略 API。通常集群管理员和集群用户不是同一个人，并且以 Pod 或任何将创建 Pod 的资源的形式创建工作负载不应该等同于“集群上的最高使用者”。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 7]
```diff
@@ -0,0 +1,250 @@
...
+upstream Kubernetes in February 2016. Now that the PSP object
+had been created, the next step was to add an admission controller that could enforce
+these policies. The first step was to add the admission
+[without taking into account the users or groups](https://github.com/kubernetes/kubernetes/pull/7893#issuecomment-180410539).
+A specific [issue to bring PodSecurityPolicy to a usable state](https://github.com/kubernetes/kubernetes/issues/23217)
+was added to keep track of the progress and a first version of the admission
+controller was merged in [pull request named PSP admission](https://github.com/kubernetes/kubernetes/pull/24600)
+in May 2016. Then around two months later, Kubernetes 1.3 was released.
+-->
+PodSecurityPolicy 的根源是[关于安全策略的第一个拉取请求](https://github.com/kubernetes/kubernetes/pull/7893)，
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:113]:
```suggestion
PodSecurityPolicy 的根源是[早期关于安全策略的一个拉取请求](https://github.com/kubernetes/kubernetes/pull/7893)，
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+merged in early November 2016 allowed administrators to use multiple policies
+in a cluster to grant different levels of access for different types of users.
+Later, a [pull request](https://github.com/kubernetes/kubernetes/pull/52849)
+merged in October 2017 fixed [a design issue](https://github.com/kubernetes/kubernetes/issues/36184)
+on ordering PodSecurityPolicies between mutating and alphabetical order, and continued to
+build the PSP admission as we know it. After that, many improvements and fixes
+followed to build the PodSecurityPolicy feature of recent Kubernetes releases.
+-->
+之后，PSP 准入控制器通过添加最初留下的内容进行了增强。[授权机制](https://github.com/kubernetes/kubernetes/pull/33080)
+2016 年 11 月上旬合并，允许管理员在集群中使用多个策略，为不同类型的用户授予不同级别的访问权限。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:143]:
```suggestion
之后，PSP 准入控制器通过添加最初留下的内容进行了增强。
於 2016 年 11 月上旬合併的[授权机制](https://github.com/kubernetes/kubernetes/pull/33080)
允许管理员在集群中使用多个策略，为不同类型的用户授予不同级别的访问权限。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 9]
```diff
@@ -0,0 +1,250 @@
...
+<!--
+The experience with PodSecurityPolicy concluded that most users care for two or three
+policies, which led to the creation of the [Pod Security Standards](/docs/concepts/security/pod-security-standards/),
+that define three policies:
+- **Privileged** - unrestricted policy.
+- **Baseline** - minimally restrictive policy, allowing the default pod
+  configuration.
+- **Restricted** - security best practice policy.
+-->
+PodSecurityPolicy 的经验得出的结论是，大多数用户关心两个或三个策略，这导致了 [Pod 安全标准](/docs/concepts/security/pod-security-standards/)的创建，它定义了三个策略：
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:200]:
```suggestion
PodSecurityPolicy 的经验得出的结论是，大多数用户关心两个或三个策略，这导致了 [Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)的创建，它定义了三个策略：
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 10]
```diff
@@ -0,0 +1,250 @@
...
+- **特权** - 不受限制的政策。
+- **基线** - 最小限制策略，允许默认 Pod 配置。
+- **限制** - 安全最佳实践策略。
+-
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:204]: Delete this.

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 11]
```diff
@@ -0,0 +1,250 @@
...
+creation of Pod Security admission, the
+[SIG auth update at KubeCon NA 2019](https://www.youtube.com/watch?v=SFtHRmPuhEw)
+and the [PodSecurityPolicy Replacement: Past, Present, and Future](https://www.youtube.com/watch?v=HsRRmlTJpls)
+presentation at KubeCon NA 2021 records are available.
+-->
+## 下一步是什么
+
+有关 SIG Auth 流程的更多详细信息，包括 PodSecurityPolicy 删除和 Pod 安全准入的创建，
+[SIG auth update at KubeCon NA 2019](https://www.youtube.com/watch?v=SFtHRmPuhEw) 
+和 [PodSecurityPolicy Replacement: Past, Present, and Future](https://www.youtube.com/watch?v=HsRRmlTJpls) 在 KubeCon NA 2021 上的演示记录可用。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:230]:
```suggestion
請參閱在 KubeCon NA 2021 的
[SIG auth update at KubeCon NA 2019](https://www.youtube.com/watch?v=SFtHRmPuhEw) 和
[PodSecurityPolicy Replacement: Past, Present, and Future](https://www.youtube.com/watch?v=HsRRmlTJpls)
議程錄影。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 12]
```diff
@@ -0,0 +1,250 @@
...
+
+<!--
+Particularly on the PSP removal, the
+[PodSecurityPolicy Deprecation: Past, Present, and Future](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)
+blog post is still accurate.
+-->
+特别是在 PSP 删除方面，[PodSecurityPolicy 弃用：过去、现在和未来](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)博客文章仍然是准确的。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:237]:
```suggestion
特别是在 PSP 移除方面，[PodSecurityPolicy 弃用：过去、现在和未来](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)博客文章仍然是准确的。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 13]
```diff
@@ -0,0 +1,250 @@
...
+<!--
+And for the new Pod Security admission,
+[documentation is available](/docs/concepts/security/pod-security-admission/).
+In addition, the blog post
+[Kubernetes 1.23: Pod Security Graduates to Beta](/blog/2021/12/09/pod-security-admission-beta/)
+along with the KubeCon EU 2022 presentation
+[The Hitchhiker's Guide to Pod Security](https://www.youtube.com/watch?v=gcz5VsvOYmI)
+give great hands-on tutorials to learn.
+-->
+对于新的 Pod 安全许可，[提供文档](/docs/concepts/security/pod-security-admission/)。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:248]:
```suggestion
对于新的 Pod 安全许可，[提供文档](/zh-cn/docs/concepts/security/pod-security-admission/)。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 14]
```diff
@@ -0,0 +1,250 @@
...
+此外，博客文章 [Kubernetes 1.23:Pod Security Graduers to Beta](/blog/2021/12/09/pod-security-admission-beta/)
+以及 KubeCon EU 2022 演示文稿 [the Hitchhicker’s Guide to Pod Security](https://www.youtube.com/watch?v=gcz5VsvOYmI) 提供了很好的实践教程。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:250]: 演示 -> 演講

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/timeline.svg [hunk 15]
```diff
@@ -0,0 +1,2008 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/timeline.svg:2]: You have the `timeline.drawio` source file, please change those English word to Mandarin and export again.

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 16]
```diff
@@ -0,0 +1,250 @@
...
+  understanding of Linux security primitives. e.g. MustRunAsNonRoot +
+  AllowPrivilegeEscalation.
+-->
+- **有缺陷的授权模式** - 如果用户在 PSP 上具有允许该 Pod 的**使用**动词，或者该 Pod 
+  的服务帐户在允许的 PSP 上具有**使用**权限，则用户可以创建一个 Pod。
+- **难以推广** - PSP 失败关闭。也就是说，在没有策略的情况下，所有 Pod 都会被拒绝。
+  这主要意味着默认情况下无法启用它，并且用户必须在启用该功能之前为所有工作负载添加 PSP，
+  因此没有提供审计模式来发现哪些 Pod 会不被新策略所允许。选择加入模式还导致测试覆盖率不足，并因跨功能不兼容而经常出现故障。
+  而且与 RBAC 不同的是，没有将 PSP 清单与项目一起运送的强大文化。
+- **不一致的无边界 API** - API 的发展有很多不一致的地方，特别是由于许多小众用例的请求：
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:186]:
```suggestion
- **不一致的无边界 API** - API 的发展有很多不一致的地方，特别是由于许多小众场景的请求：
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 17]
```diff
@@ -0,0 +1,250 @@
...
+  如标签、调度、细颗粒的音量控制等。它的可组合性很差，有一个弱的优先级模型，导致了意外的突变优先级。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:187]:
```suggestion
  如标签、调度、细粒度的卷控制等。它的可组合性很差，优先级模型较弱，会导致意外的变更优先级。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 18]
```diff
@@ -0,0 +1,250 @@
...
+  这使得 PSP 与其他第三方准入控制器的结合真的很困难。
+- **需要安全知识** - 有效的使用仍然需要了解 Linux 的安全原语。例如：MustRunAsNonRoot + AllowPrivilegeEscalation。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:189]:
```suggestion
- **需要安全知识** - 有效使用 PSP 仍然需要了解 Linux 的安全原语。
  例如：MustRunAsNonRoot + AllowPrivilegeEscalation。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md (hunk 9 above)
Comment (tengqm) [reply content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:200]: 适当折行

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 19]
```diff
@@ -0,0 +1,250 @@
...
+- 
+<!--
+The replacement for PSP, the new [Pod Security Admission](/docs/concepts/security/pod-security-admission/)
+is an in-tree, stable for Kubernetes v1.25, admission plugin to enforce these
+standards at the namespace level. It makes it easier to enforce basic pod
+security without deep security knowledge. For more sophisticated use cases, you
+might need a third-party solution that can be easily combined with Pod Security
+Admission.
+-->
+PSP 的替代品，新的 [Pod 安全准入](/docs/concepts/security/pod-security-admission/)
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:213]:
```suggestion
作为 PSP 的替代品，新的 [Pod 安全准入](/zh-cn/docs/concepts/security/pod-security-admission/)
```

**Review by Michelle951 — COMMENTED** [review]
/lgtm

**Comment by tengqm** [comment]
/hold
I'm disappointed by this revision. The author either ignores the feedback, or revises the text in different ways than suggested while mark the feedbacks addressed. This is a waste of time. For me, it means more than 30 minutes.

**Comment by sarazqy** [comment]
I'm very sorry to waste your time. I'll check carefully; I wonder why it didn't work this time. The reason is that I refer to the `Files changed` page every time I make changes, and this time a line of code has the same opinion, it will only display one in order, some I have not seen your opinion, I will be more serious later ,terribly sorry. @tengqm

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 20]
```diff
@@ -0,0 +1,264 @@
...
+<!--
+The origin of the creation of PodSecurityPolicy is difficult to track, notably
+because it was mainly added before Kubernetes Enhancements Proposal (KEP)
+process, when design proposals were still a thing. Indeed, the archive of the final
+[design proposal](https://github.com/kubernetes/design-proposals-archive/blob/main/auth/pod-security-policy.md)
+is still available. Nevertheless, a [KEP issue number five](https://github.com/kubernetes/enhancements/issues/5) 
+was created after the first pull requests were merged.
+-->
+PodSecurityPolicy 的创建起源很难追踪，特别是因为它主要是在 Kubernetes 增强提案 (KEP) 流程之前添加的，
+当时仍在使用设计提案（Design Proposal）。事实上，最终 [设计提案](https://github.com/kubernetes/design-proposals-archive/blob/main/auth/pod-security-policy.md)的存档仍然可以找到。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:73]:
```suggestion
当时仍在使用设计提案（Design Proposal）。事实上，最终[设计提案](https://github.com/kubernetes/design-proposals-archive/blob/main/auth/pod-security-policy.md)的存档仍然可以找到。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 21]
```diff
@@ -0,0 +1,264 @@
...
+and the [PodSecurityPolicy Replacement: Past, Present, and Future](https://www.youtube.com/watch?v=HsRRmlTJpls)
+presentation at KubeCon NA 2021 records are available.
+-->
+## 下一步是什么
+
+有关 SIG Auth 流程的更多详细信息，包括 PodSecurityPolicy 删除和 Pod 安全准入的创建，
+请参阅在 KubeCon NA 2021 的
+[SIG auth update at KubeCon NA 2019](https://www.youtube.com/watch?v=SFtHRmPuhEw) 和 
+[PodSecurityPolicy Replacement: Past, Present, and Future](https://www.youtube.com/watch?v=HsRRmlTJpls) 
+课程录影。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:244]: It's a presentation, not an online course.

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 22]
```diff
@@ -0,0 +1,264 @@
...
+And for the new Pod Security admission,
+[documentation is available](/docs/concepts/security/pod-security-admission/).
+In addition, the blog post
+[Kubernetes 1.23: Pod Security Graduates to Beta](/blog/2021/12/09/pod-security-admission-beta/)
+along with the KubeCon EU 2022 presentation
+[The Hitchhiker's Guide to Pod Security](https://www.youtube.com/watch?v=gcz5VsvOYmI)
+give great hands-on tutorials to learn.
+-->
+对于新的 Pod 安全许可，[提供文档](/zh-cn/docs/concepts/security/pod-security-admission/)。
+此外，博客文章 [Kubernetes 1.23:Pod Security Graduers to Beta](/blog/2021/12/09/pod-security-admission-beta/)
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:263]:
```suggestion
此外，博客文章 [Kubernetes 1.23: Pod Security Graduers to Beta](/blog/2021/12/09/pod-security-admission-beta/)
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 23]
```diff
@@ -0,0 +1,264 @@
...
+以及 KubeCon EU 2022 演讲文稿 [the Hitchhicker’s Guide to Pod Security](https://www.youtube.com/watch?v=gcz5VsvOYmI) 提供了很好的实践教程。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:264]: Not sure what's the "文稿" means here.

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 24]
```diff
@@ -0,0 +1,265 @@
...
+the form of a Pod or any resource that will create a Pod should not equal being
+"root on the cluster". It could also encourage best practices by configuring
+more secure defaults through mutation and decoupling low-level Linux security
+decisions from the deployment process.
+-->
+PodSecurityPolicy 与其他专门的准入控制插件一样，在与 Pod 安全设置有关的特定字段上提供细粒度权限，
+作为内置策略 API。通常集群管理员和集群用户不是同一个人，并且以 Pod 形式或任何将创建
+
+它还可以通过变更配置来应用更安全的默认值，并将底层 Linux 安全决策与部署过程分离来促进最佳实践。
+Pod 的资源的形式创建工作负载的权限不应该等同于“集群上的 root 账户”。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:51]: 这一段重新组织。

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md (hunk 6 above)
Comment (sarazqy) [reply content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md]: > 

thanks for checking, “集群上的 root 账户” looks better

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 25]
```diff
@@ -0,0 +1,264 @@
...
+以跟踪进展，并在[名为 PSP 准入的拉取请求](https://github.com/kubernetes/kubernetes/pull/24600)中合并了准入控制器的第一个版本。
+然后大约两个月后，发布了 Kubernetes 1.3。
+
+
+<!--
+Here is a timeline that recaps the main pull requests of the birth of the
+PodSecurityPolicy and its admission controller with 1.0 and 1.3 releases as
+reference points.
+-->
+这是一个时间表，它以 1.0 和 1.3 版本作为参考点，回顾了 PodSecurityPolicy 及其准入控制器诞生的主要拉取请求。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:134]:
```suggestion
下面是一个时间表，它以 1.0 和 1.3 版本作为参考点，回顾了 PodSecurityPolicy 及其准入控制器诞生的主要拉取请求。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 26]
```diff
@@ -0,0 +1,264 @@
...
+merged in early November 2016 allowed administrators to use multiple policies
+in a cluster to grant different levels of access for different types of users.
+Later, a [pull request](https://github.com/kubernetes/kubernetes/pull/52849)
+merged in October 2017 fixed [a design issue](https://github.com/kubernetes/kubernetes/issues/36184)
+on ordering PodSecurityPolicies between mutating and alphabetical order, and continued to
+build the PSP admission as we know it. After that, many improvements and fixes
+followed to build the PodSecurityPolicy feature of recent Kubernetes releases.
+-->
+之后，PSP 准入控制器通过添加最初被搁置的内容进行了增强。
+在 2016 年 11 月上旬合并[授权机制](https://github.com/kubernetes/kubernetes/pull/33080)，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:150]:
```suggestion
在 2016 年 11 月上旬合并[鉴权机制](https://github.com/kubernetes/kubernetes/pull/33080)，
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 27]
```diff
@@ -0,0 +1,264 @@
...
+- **Privileged** - unrestricted policy.
+- **Baseline** - minimally restrictive policy, allowing the default pod
+  configuration.
+- **Restricted** - security best practice policy.
+-->
+PodSecurityPolicy 的经验得出的结论是，大多数用户关心两个或三个策略，这导致了
+[Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)的创建，它定义了三个策略：
+- **特权** - 不受限制的政策。
+- **基线** - 最小限制策略，允许默认 Pod 配置。
+- **限制** - 安全最佳实践策略。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:214]:
```suggestion
- **Privileged（特权的）** - 策略不受限制。
- **Baseline（基线的）** - 策略限制很少，允许默认 Pod 配置。
- **Restricted（受限的）** - 安全最佳实践策略。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 28]
```diff
@@ -0,0 +1,264 @@
...
+<!--
+The replacement for PSP, the new [Pod Security Admission](/docs/concepts/security/pod-security-admission/)
+is an in-tree, stable for Kubernetes v1.25, admission plugin to enforce these
+standards at the namespace level. It makes it easier to enforce basic pod
+security without deep security knowledge. For more sophisticated use cases, you
+might need a third-party solution that can be easily combined with Pod Security
+Admission.
+-->
+作为 PSP 的替代品，新的 [Pod 安全准入](/zh-cn/docs/concepts/security/pod-security-admission/)
+是 Kubernetes v1.25 的树内稳定的准入插件，用于在命名空间级别强制执行这些标准。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:225]:
```suggestion
作为 PSP 的替代品，新的 [Pod 安全准入](/zh-cn/docs/concepts/security/pod-security-admission/)是
Kubernetes v1.25 的树内稳定的准入插件，用于在命名空间级别强制执行这些标准。 
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 29]
```diff
@@ -0,0 +1,264 @@
...
+The replacement for PSP, the new [Pod Security Admission](/docs/concepts/security/pod-security-admission/)
+is an in-tree, stable for Kubernetes v1.25, admission plugin to enforce these
+standards at the namespace level. It makes it easier to enforce basic pod
+security without deep security knowledge. For more sophisticated use cases, you
+might need a third-party solution that can be easily combined with Pod Security
+Admission.
+-->
+作为 PSP 的替代品，新的 [Pod 安全准入](/zh-cn/docs/concepts/security/pod-security-admission/)
+是 Kubernetes v1.25 的树内稳定的准入插件，用于在命名空间级别强制执行这些标准。 
+无需深入的安全知识，就可以更轻松地实施基本的 Pod 安全性。对于更复杂的用例，你可能需要一个可以轻松与 Pod 安全准入结合的第三方解决方案。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:226]:
```suggestion
无需深入的安全知识，就可以更轻松地实施基本的 Pod 安全性。
对于更复杂的用例，你可能需要一个可以轻松与 Pod 安全准入结合的第三方解决方案。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 30]
```diff
@@ -0,0 +1,264 @@
...
+and the [PodSecurityPolicy Replacement: Past, Present, and Future](https://www.youtube.com/watch?v=HsRRmlTJpls)
+presentation at KubeCon NA 2021 records are available.
+-->
+## 下一步是什么
+
+有关 SIG Auth 流程的更多详细信息，包括 PodSecurityPolicy 删除和 Pod 安全准入的创建，
+请参阅在 KubeCon NA 2021 的
+[SIG auth update at KubeCon NA 2019](https://www.youtube.com/watch?v=SFtHRmPuhEw) 和 
+[PodSecurityPolicy Replacement: Past, Present, and Future](https://www.youtube.com/watch?v=HsRRmlTJpls) 
+演示录影。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:243]:
```suggestion
演示录像。
```

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 31]
```diff
@@ -0,0 +1,264 @@
...
+<!--
+And for the new Pod Security admission,
+[documentation is available](/docs/concepts/security/pod-security-admission/).
+In addition, the blog post
+[Kubernetes 1.23: Pod Security Graduates to Beta](/blog/2021/12/09/pod-security-admission-beta/)
+along with the KubeCon EU 2022 presentation
+[The Hitchhiker's Guide to Pod Security](https://www.youtube.com/watch?v=gcz5VsvOYmI)
+give great hands-on tutorials to learn.
+-->
+对于新的 Pod 安全许可，[提供文档](/zh-cn/docs/concepts/security/pod-security-admission/)。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:261]:
```suggestion
对于新的 Pod 安全许可，[可以访问文档](/zh-cn/docs/concepts/security/pod-security-admission/)。
```

**Pull request merged**